t/apicast-policy-content-caching.t

```diff
@@ -429,3 +429,103 @@ __DATA__
 ["X-Cache-Status: MISS", "X-Cache-Status: MISS"]
 --- no_error_log
 [error]
+
+
+
+=== TEST 8: HEAD request is enabled if no content-caching
```

Comment:
Would be good to add a brief explanation here or in `TEST 9` that clarifies that this is tested because of https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_convert_head